contracts/VaultV3.vy

```diff
@@ -80,6 +82,9 @@ event DebtUpdated:
 event UpdateAccountant:
     accountant: address
 
+event UpdateQueueManager:
+    queue_manager: address
```

Comment:
should it be indexed?

Reply:
yes. All the events need to be reviewed and indexes added

Reply:
https://github.com/flashfish0x/yearn-vaults-v3/pull/1/commits/13f0152deac86c0041ddce48f97557f0709f58e6